### ISSUE TITLE
Android: Color-blind correction/Color-blind Filters

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30872]
## Description
Apps will very rarely need to know if this is enabled, as it works at the system drawing layer above the apps themselves, so there should not be any cases where this doesn't work correctly.

## React Native version:
v0.63

**Comment by blavalla** [comment]
Google doesn't provide an API for this, so while it's technically possible to query using something like reflection, I don't think it's worth adding the hacky code required to the codebase, especially given it's relatively limited use cases. 

Closing out.
